File: C# Activities/TestApp/Array1 Activitie1.cs
using System;

class Array1Activitie1
{
   static void Main() 
    {
        int[] precios = new int[3]; // c declara el arreglo

        for (int i = 0; i < 3; i++) //el int c iniciliaza en 0 y si es menor a 3 se incrementa en 1
        {
            Console.Write($"Dame el precio {i + 1}: "); // c escribe en la consolsa
            precios[i] = Convert.ToInt32(Console.ReadLine()); //el valor lo convierte en un entero y lo guarda en el arreglo
        }

        Console.WriteLine("\nLos precios que ingresaste son:"); //c escribe en la consola
        for (int i = 0; i < 3; i++) //el int c iniciliaza en 0 y si es menor a 3 se incrementa en 1
        {
            Console.WriteLine(precios[i]); //c escribe en la consola el valor guardado en el arreglo
        }

        Console.WriteLine("\nPresiona cualquier tecla para salir...");
        Console.ReadKey(); // siempre va este codigo para que no se cierre la consola
    }
 }

File: Activities C++/output/ActividadesArraysEstructura/Array1 Activitie1.cpp
#include <iostream> // Esta libreria siempre va
using namespace std; // Este tambien siempre

int main() { // De entrada
    int precios[3]; // Declaramos un arreglo de 3 elementos

    for (int i = 0; i < 3; i++) {  //ciclo para llenar el arreglo
        cout << "Dame el precio " << i + 1 << ": "; // Pedimos el precio
        cin >> precios[i]; // El usuario ingresa el precio y se guarda en el arreglo
    }

    cout << "\nLos precios que ingresaste son:\n";
    for (int i = 0; i < 3; i++) { // Ciclo para mostrar los precios
        cout << precios[i] << endl ; // Mostramos el precio
    }

    return 0;
}